### PR TITLE
fix mobile responsiveness & header height, adjust opacity scrolling

### LINKED ISSUE #3: mobile resize is bad (need meta tag)

### LINKED ISSUE #4: Header fade happens too early

### PR DESCRIPTION
**Pull request by jesseditson**
Fixes #3 
Fixes #4

### CODE REVIEW

File: components/Jumbo.js [hunk 1]
```diff
@@ -1,33 +1,54 @@
 import Button from './Button'
 
+const MIN_OPACITY = 0.1
+const OPACITY_START = 0.4
+
 export default ({top}) => {
-  const opacity = 1 - top / 800
+  const height =
+    typeof document === 'undefined'
```
Comment (arv) [inline components/Jumbo.js:8]: when is document ever undefined?

File: components/Jumbo.js [hunk 2]
```diff
@@ -1,33 +1,54 @@
...
+      ? 400
+      : document.getElementById('jumbo').getBoundingClientRect().height
+  // Start fade at OPACITY_START % of total height
+  const min = height * OPACITY_START
+  const completion = Math.max((top - min) / (height - min), 0)
+  // Interpolate value between 1 and MIN_OPACITY
+  const opacity = 1 - (1 - MIN_OPACITY) * completion
+  console.log(completion, opacity)
```
Comment (arv) [inline components/Jumbo.js:16]: log

Comment (jesseditson) [reply components/Jumbo.js:16]: Yeah, pushed a fix directly to master 😬

File: components/Jumbo.js (hunk 1 above)
Comment (jesseditson) [reply components/Jumbo.js:8]: next.js runs both on the server and client - so you don't have a guaranteed document/window/etc. In our case, we use `export`, which will render everything with node when building, which is the specific case this is covering.

Comment (arv) [reply components/Jumbo.js:8]: Thanks.

**Pull request merged**